pymatgen/entries/compatibility.py

```diff
@@ -689,10 +989,54 @@ def __init__(self, compat_type="Advanced", correct_peroxide=True,
         self.check_potcar_hash = check_potcar_hash
         fp = os.path.join(MODULE_DIR, "MPCompatibility.yaml")
         super().__init__(
-            [PotcarCorrection(MPRelaxSet, check_hash=check_potcar_hash),
-             GasCorrection(fp),
-             AnionCorrection(fp, correct_peroxide=correct_peroxide),
-             UCorrection(fp, MPRelaxSet, compat_type)])
+            [
+                PotcarCorrection(MPRelaxSet, check_hash=check_potcar_hash),
+                GasCorrection(fp),
+                AnionCorrection(fp, correct_peroxide=correct_peroxide),
+                UCorrection(fp, MPRelaxSet, compat_type),
+            ]
+        )
+
+
+class MaterialsProject2020Compatibility(CorrectionsList):
+    """
+    This class implements the GGA/GGA+U mixing scheme, which allows mixing of
+    entries. Note that this should only be used for VASP calculations using the
+    MaterialsProject parameters (see pymatgen.io.vaspio_set.MPVaspInputSet).
+    Using this compatibility scheme on runs with different parameters is not
+    valid.
+    """
+
+    def __init__(
+        self, compat_type="Advanced", correct_peroxide=True, check_potcar_hash=False
```

Comment:
I'd advise against, considering how much confusion continues to be caused by the hashing checks we added to `InputSet`. We could certainly turn it on internally for purposes of building our database though. I haven't given this a lot of thought; we default to `False` because that's what the old MPCompatibility does.